src/screens/SearchScreen.js

```diff
@@ -105,23 +105,6 @@ class SearchScreen extends Component {
         }
     };
 
-    selectedSiteIsFavorite = () => {
-        const {currentUser, selectedSite} = this.props;
-
-        return !!_.find(currentUser.favorites, favorite => favorite.id === selectedSite.id);
-    };
-
-    toggleSiteFavorite = () => {
-        const {currentUser, selectedSite} = this.props;
-
-        if (!this.selectedSiteIsFavorite()) {
-            this.props.attemptToAddFavorite({selectedSite, currentUser});
-        } else {
-            this.props.attemptToRemoveFavorite({selectedSite, currentUser});
-        }
-
-    };
-
```

Comment:
No longer needed in the `SearchScreen` as we now use it directly in the `SitePreview` component.